src/en/help-oracle.md

```diff
@@ -1,6 +1,6 @@
-Title: Using the Oracle cloud
+Title: Using Oracle GCE with Juju
```

Comment:
should probably be *Oracle Compute with Juju*

Reply:
Agreed.